js/clone.js

```diff
@@ -156,8 +156,14 @@
 	 * @param $container .rwmb-input container
 	 */
 	function toggleRemoveButtons( $container ) {
-		var $clones = $container.children( '.rwmb-clone' );
-		$clones.children( '.remove-clone' ).toggle( $clones.length > 1 );
+
+		var $clones = $container.children( '.rwmb-clone' ),
+		    minClone = 1;
+	
+		if($container.data( 'min-clone' )){
```

Comment:
Can you add spaces after brackets `(` and before brackets `)` to follow WP coding standards?

Reply:
Yes of course!